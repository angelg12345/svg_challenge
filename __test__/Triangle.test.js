const Triangle = require('../lib/Triangle');

test('renders a triangle', () => {
  const triangle = new Triangle('blue', 'srt', 'white');
  expect(triangle.render()).toMatchSnapshot();
});
