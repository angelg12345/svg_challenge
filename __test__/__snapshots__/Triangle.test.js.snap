// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a triangle 1`] = `
" <svg height="500" width="500" xmlns="http://www.w3.org/2000/svg">
		    <polygon points="250,60 100,400 400,400" fill= "blue" />
            <text fill="white" font-size="70" x= "215" y = "230">srt</text>
	  </svg>"
`;
